package.json

```diff
@@ -1,8 +1,6 @@
 {
   "name": "opentype.js",
   "description": "OpenType font parser",
-  "version": "2.0.0",
-  "sideEffects": false,
```

Comment:
we need the ``"sideEffects": false,`` entry to allow bundlers to optimize the package.

Reply:
I thought it was a webpack-specific flag (which is not used anymore)

I've added it back :+1:

Reply:
I can't find where the version and side effects properties went?

Reply:
the version is set by github action upon git tag
See https://github.com/yne/opentype.yne.js/blob/master/.github/workflows/release.yml#L22

the name is also automatically so anybody can fork and rename they fork so they can publish they own fork without conflicting when PR

Reply:
But the sideEffects?

Reply:
it shall be before eslintConfig
it is not ?

Reply:
Oh boy... Yes of course it is. I must have missed it in the compact diff view, but I can see it when I open the full package json file. Sorry for the confusion!